charts/pega/templates/pega-tier-deployment.yaml

```diff
@@ -3,7 +3,7 @@
 {{ range $index, $dep := .Values.global.tier}}
 {{ if ($dep.volumeClaimTemplate) }}
 {{ $kindName = "StatefulSet" }}
-{{ $apiVer = "apps/v1beta2" }}
+{{ $apiVer = "apps/v1" }}
 {{ else }}
 {{ $kindName = "Deployment" }}
 {{ $apiVer = "extensions/v1beta1" }}
```

Comment:
Should we also modify the api for Deployment to use apps/v1 as per kubernetes documentation https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/

Reply:
I tested this and it wouldn't work without this change. So I guess this should be changed.

```yaml
{{ $kindName = "Deployment" }}
{{ $apiVer = "apps/v1" }}
```

Reply:
Fixed.  Good find!